config/params/config.go

```diff
@@ -470,6 +469,10 @@ func (ns *NetworkSchedule) LastFork() NetworkScheduleEntry {
 	return ns.entries[0]
 }
 
+func (ns *NetworkSchedule) lastEntry() NetworkScheduleEntry {
+	return ns.entries[len(ns.entries)-1]
+}
+
```

Comment:
Although it may be an invalid schedule, i think this would panic if there were no entries. Want to add a nil / len == 0 check?